Description: Fix docutils deprecated function
Author: Jean Baptiste Favre <jbfavre@debian.org>
Origin: other
Forwarded: https://github.com/apache/trafficserver/pull/9886
Reviewed-by: Jean Baptiste Favre <jbfavre@debian.org>
Last-Update: 2023-01-30
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/doc/developer-guide/documentation/adding-domains.en.rst
+++ b/doc/developer-guide/documentation/adding-domains.en.rst
@@ -209,7 +209,7 @@ Our variables domain might have the foll
         title['first'] = False
         title['objtype'] = node['objtype']
         self.add_name(title)
-        title.set_class('ts-variable-title')
+        title['classes'] = 'ts-variable-title'
 
         title += sphinx.addnodes.desc_name(var_name, var_name)
         node.append(title)
